CMakeLists.txt

```diff
@@ -89,11 +89,14 @@ if(PACKAGE_MODE)
     INSTALL(FILES "${CMAKE_SOURCE_DIR}/bin/docs/PCSX2.1"            DESTINATION "${CMAKE_INSTALL_MANDIR}/man1")
     if(NOT DISABLE_PCSX2_WRAPPER)
         INSTALL(FILES "${CMAKE_SOURCE_DIR}/linux_various/PCSX2-linux.sh"          DESTINATION "${CMAKE_INSTALL_BINDIR}")
+        INSTALL(FILES "${CMAKE_SOURCE_DIR}/pcsx2/Docs/portable.yaml"              DESTINATION "${CMAKE_INSTALL_BINDIR}")
```

Comment:
Yeah so. It's a bit of a change. I set portable mode to true by default. After first time wizard that gets set to false. Since portable.yaml is a tracked file that would be seen as a change in git. To avoid that I moved the portable file to where it wouldn't be used and am having cmake copy it into bin as a non tracked changeable file